README.md

```diff
@@ -30,28 +30,111 @@ pip install -r requirements-cuda.txt
+
+## Validation Steps
+
+After completing the setup process, ensure that everything is working correctly:
+
+1. **Verify Hugging Face Hub Login**:
+   - Run the following command to verify that you are logged in to the Hugging Face Hub:
+     ```bash
+     huggingface-cli whoami
+     ```
+   - If your username is displayed, it means you are successfully logged in.
+
+2. **Check Dataset Access**:
+   - Visit the [HuggingFace Datasets - ImageNet-1k](https://huggingface.co/datasets/imagenet-1k) page.
+   - Make sure you can view and access the dataset details without any permission errors.
+
+3. **Accept Dataset Access (If Required)**:
+   - If you encounter any permission errors while accessing the ImageNet-1k dataset, ensure that you follow the instructions provided on the dataset page to grant access.
 
-1. Create a HuggingFace account from <https://huggingface.co/>
-2. Create a User Access Token by following steps in <https://huggingface.co/docs/hub/security-tokens>
-3. Download the `huggingface_hub` library and login through `huggingface-cli` with your User Access Token by following the steps in <https://huggingface.co/docs/huggingface_hub/quick-start>
 
-#### Benchmarking Datasets Setup
+### Benchmarking Datasets Setup
 
-HuggingFace datasets will download into `${HF_HOME}/datasets/` once HuggingFace Hub access is setup.
 
-COCO: is downloaded automatically from <https://cocodataset.org>, no login is needed. The cache location is `~/.cache/coco`
+To set up the three required datasets for running benchmarking tests within this repository, follow these steps for each dataset:
 
-LGG Segmentation Dataset: must be manually downloaded from <https://www.kaggle.com/datasets/mateuszbuda/lgg-mri-segmentation>, this requires a Kaggle login. The commands below are to extract the downloaded archive into the correct location: `~/.cache/mldata/lgg_segmentation/kaggle_3m`.
+1. **HuggingFace datasets**: will download into `${HF_HOME}/datasets/` once [HuggingFace Hub access](#setup-access-to-huggingface-hub) is set up.
+
+2. **COCO Dataset**: You can automatically download the COCO dataset from [here](https://cocodataset.org/#download:~:text=2017%20Val%20images%20%5B5K/1GB%5D). No login is required, and the dataset will be cached in `~/.cache/coco`.
+
+To download the COCO dataset, follow these steps:
 
 ```bash
 # use another location for MLDATA_DIR if desired, below is default
+# Create the `coco` directory inside the cache directory:
+mkdir -p ~/.cache/mldata/coco
+
+# Navigate to the `coco` directory:
+cd ~/.cache/coco
+
+# Create the `images` directory:
+mkdir images
+cd images
+
+# Download the COCO validation images:
+wget https://cocodataset.org/#download:~:text=2017%20Val%20images%20%5B5K/1GB%5D
+
+# Unzip the downloaded file:
+unzip val2017.zip
+
+# Move back to the `coco` directory:
+cd ..
+
+# Create the `annotations` directory:
+mkdir annotations
+cd annotations
```

Comment:
don't need this -- delete

Reply:
without the creation of this annotations folder , the wget http://images.cocodataset.org/zips/val2017.zip will download as val2017 after its unzip. But we need it inside the annotations folder .

Reply:
deleted here https://github.com/tenstorrent/benchmarking/commits/vcanic/benchmark_release_feb23/#:~:text=Mar%2015%2C%202024-,deleting,-anirudTT